.github/pull_request_template.md

```diff
@@ -1,10 +1,38 @@
-This PR implements the following **changes:**
+## summary
 
-* insert your summary of your request here, add more bullets as needed
+Basic description of work done.
 
+[Link to Preview]()
 
-**URL / Link to page**
+## steps to recreate
 
-<!-- Link to the page that will be changed -->
-<!-- Delete this section if not needed -->
+## solution
 
+### markdown changes
+### features/enhancements/bugs
+
+- [ ] item 1
+- [ ] item 2
+
+---
+
+## dev check list
+
+### a11y
+- [ ] voice over, screen reader tested
+- [ ] color contrast 
+- [ ] run axe tools
+
+### markup
+- [ ] semantic tags
+- [ ] hierarchy structure
+- [ ] run prettier
+
+### styles
+- [ ] bemified?
+- [ ] use uswds framework
+- [ ] run prettier?
+
+### github
+- [ ] review files tab
+- [ ] assign to self, request review
```

Comment:
We can use comments so that they're reminders and not necessarily task items OR we could simplify this:

```md
## Dev checklist

- [ ] Basic a11y testing (contrast, scanning, voice over)
- [ ] Meets WCAG 2.0
- [ ] Validated HTML
- [ ] Code is properly formatted
```